doc/releases/changelog-dev.md

```diff
@@ -45,7 +54,7 @@
   2: ──RX(1.5)──Rot(0.401, 0.0795, 0.731)──RY(1.15)──Rot(0.756, 0.38, 0.38)─────RX(-1.8)──┤
   ```
 
-  Applying the `qml.fourier.advanced_spectrum` function to the circuit for the non-trainable
+  Applying the `qml.fourier.qnode_spectrum` function to the circuit for the non-trainable
```

Comment:
@dwierichs I can't seem to comment on it, but in the codeblock below, I think we should have
```pycon
>>> spec = qml.fourier.qnode_spectrum(circuit,...)
```
to match the text description here :)